test/functional/ui/045_folding_spec.lua

```diff
@@ -0,0 +1,221 @@
+-- Tests for folding.
+local Screen = require('test.functional.ui.screen')
```

Comment:
Legacy tests that were reworked still live in legacy. The file shouldn't be moved/renamed.